spec/integration/spec_file_load_errors_spec.rb

```diff
@@ -54,6 +54,37 @@
     EOS
   end
 
+  it 'prints a single error when it happens on --require files' do
+    write_file_formatted "helper_with_error.rb", "raise 'boom'; class Perry; end"
```

Comment:
Would prefer the file just contain the raise here.

Reply:
The thing is that in order to make this a proper regression spec, I need to make the spec file loading dependent on the helper. Otherwise the spec will pass without my changes too. I don't think I can do that with just a raise...

Reply:
Well the issue is that we carry on if the helper errors at all, not that it's dependant, if you had just a `raise` in your spec helper that'd still get lost, and would cause other issues, for example if you were depending on your spec helper to clean the db and it raised and RSpec carrys on you'd get failures in a different way.

Reply:
Right, so in order to make this a proper regression spec I can maybe add a raise to the spec?

Reply:
No you just need to assert that no examples are run

Reply:
No examples will be run either way, what I'm trying to change here is that the examples won't load either.

Reply:
Examples loaded would be displayed

Reply:
Sorry, where exactly is this information displayed? There was the "No examples found." part at the end of the output, but this is the information you're telling me to remove from the output in the other comment.

Reply:
The only difference in the output is the number of errors: "<n> errors ocurred outside of exampes" and the errors themselves. I'm not sure how I can check the behavior I'm changing (show a single error instead of 2) without actually triggering the errors :man_shrugging:.

Reply:
See below and then remove the class def?